backend/z_index.js

```diff
@@ -36,6 +37,26 @@ app.get('/absences', async (req, res) => {
   const absences = await getAbsenceForms();
   res.json(absences);
 })
+app.get('/meetingHistory', async (req,res) => {
+  const meetingHist = await getMeetingHistory();
+  res.json(meetingHist);
+})
+app.post('/updateDatabases', (req, res) => {
+  if (req.body == undefined) {
+    return res.json({
+      msg: "Error: parameter not defined",
+      data: {}
+    });
+  }
+  const body = req.body;
+  const meetingType = body.type;
+  const meetingNumber = body.number;
+  updateMembersData(meetingType, meetingNumber);
+  return res.json ({
```

Comment:
Will the success status be dependent on if the updateMembersData being successful?